client/my-sites/plans/jetpack-plans/storage-tier-upgrade/style.scss

```
@@ -0,0 +1,18 @@
+.storage-tier-upgrade {
+	display: grid;
+	grid-template-columns: repeat( auto-fit, minmax( 300px, 1fr ) );
+	gap: 24px 16px;
+
+	margin-top: 48px;
+	margin-bottom: 64px;
+
+	.jetpack-product-card__product-name {
+		font-size: 3.375rem;
+		margin-bottom: 9px;
+	}
```

* We should try to use Calypso's SASS typography variables when we can. In this case, `3.375rem` matches up to `$font-headline-large`.
* In design mock-ups, letter spacing is decreased by 2px to make the header read more smoothly.
* Header text is also center-aligned in our mock-up.

```suggestion
	.jetpack-product-card__product-name {
		font-size: $font-headline-large;
		letter-spacing: -2px;
		text-align: center;
		margin-bottom: 9px;
	}
```